cmake/templates/config_defines_entries_for_modules.cpp.in

```diff
@@ -8,7 +8,7 @@
 
 #include <hpx/modules/config_registry.hpp>
 
-namespace hpx { namespace @module_name@ {
+namespace hpx { namespace __@module_name@ {
```

Comment:
Good idea! Should we put the underscores at the end though, just to avoid any potential clash with standard library names?

Reply:
I can do that.